cli/commands/preview/list.ts

```diff
@@ -27,11 +32,18 @@ export async function runCommand(
 		}
 
 		logger.reportSuccess(
-			sprintf( _n( 'Found %d preview', 'Found %d previews', snapshots.length ), snapshots.length )
+			sprintf(
+				_n( 'Found %d preview site', 'Found %d preview sites', snapshots.length ),
+				snapshots.length
+			)
 		);
```

Comment:
Not directly related to the main changes in this PR, but I thought spelling out "preview site/-s" was more idiomatic than calling them just "preview".